db/migrate/20220705162815_add_users_ref_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsersRefToLikes < ActiveRecord::Migration[7.0]
+    def change
+      add_reference :likes, :user, null: false, foreign_key: true
```

Comment:
- [ ] Your likes table uses `user_id`, whereas you have `author_id`. Kindy fix that by referencing `author`, instead of `user`. You can use the `to_table` option to refer Rails to the users' table.

Reply:
Thank you @julie-ify 
I have updated it to author